app/src/main/java/dev/arkbuilders/arkretouch/data/model/ImageDefaults.kt

```diff
@@ -1,9 +1,15 @@
-package dev.arkbuilders.arkretouch.data
+package dev.arkbuilders.arkretouch.data.model
 
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.IntSize
 import kotlinx.serialization.Serializable
 
+/**
+ * We use serialization for these classes because we write them to storage as JSON text,
+ * To convert them correctly we use [kotlinx.serialization.encodeToString] and [kotlinx.serialization.decodeFromString] which requires an object to be [@Serializable]
+ * Refer here [dev.arkbuilders.arkretouch.data.OldStorageRepository.persistDefaults]
+*/
+
```

Comment:
@kirillt double-check

Reply:
That is a justification to not to refactor ```persistDefaults```. I will remove it if nobody has an opinion